build_runner/lib/src/server/hot_reload_client/client.dart

```diff
@@ -128,23 +141,24 @@ List<K> keys<K, V>(JsMap<K, V> map) {
   return List.from(_jsArrayFrom(map.keys()));
 }
 
-Future<Module> _reloadModule(String moduleId) {
-  var completer = Completer<Module>();
-  dartLoader.forceLoadModule(
-      moduleId,
-      allowInterop((HotReloadableModule module) =>
-          completer.complete(ModuleWrapper(module))));
-  return completer.future;
-}
-
-Future<Module> _loadModule(String moduleId) {
-  var completer = Completer<Module>();
-  dartLoader.loadModule(
-      moduleId,
-      allowInterop((HotReloadableModule module) =>
-          completer.complete(ModuleWrapper(module))));
-  return completer.future;
-}
+Future<Module> Function(String) _wrapLoaderFunction(
```

Comment:
Can you use a more descriptive name here? Something that indicates the purpose of the wrapping.